src/Lykke.Service.Stellar.Api/Startup.cs

```diff
@@ -105,139 +134,66 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
                 });
                 app.UseStaticFiles();
 
-                appLifetime.ApplicationStarted.Register(() => StartApplication().GetAwaiter().GetResult());
-                appLifetime.ApplicationStopping.Register(() => StopApplication().GetAwaiter().GetResult());
-                appLifetime.ApplicationStopped.Register(() => CleanUp().GetAwaiter().GetResult());
+                appLifetime.ApplicationStarted.Register(StartApplication);
+                appLifetime.ApplicationStopping.Register(StopApplication);
+                appLifetime.ApplicationStopped.Register(CleanUp);
             }
             catch (Exception ex)
             {
-                Log?.WriteFatalErrorAsync(nameof(Startup), nameof(Configure), "", ex).GetAwaiter().GetResult();
+                Log?.Critical(ex);
                 throw;
             }
         }
 
-        private async Task StartApplication()
+        private void StartApplication()
```

Comment:
Why this method has been made sync?

Reply:
We use it synchronously anyway:
`appLifetime.ApplicationStarted.Register(() => StartApplication().GetAwaiter().GetResult());`
so, I considered it to be a bit less messy if the method was synchronous by itself using `GetAwaiter().GetResult()` inside:
`appLifetime.ApplicationStarted.Register(StartApplication);`

Reply:
See #3